torch/csrc/jit/passes/quantization.cpp

```diff
@@ -148,6 +148,37 @@ bool valueNeedsToBeQuantized(Value* v) {
   return false;
 }
 
+script::Module findChildModule(
+    const script::Module& module,
+    const std::vector<std::string>& path) {
+  script::Module m = module;
+  for (const auto& p : path) {
+    m = m.attr(p).toModule();
+  }
+  return m;
+}
+
+// Check if value is the input of the graph
+bool hitGraphInput(Value* value) {
+  Graph* graph = value->owningGraph();
+  const auto& inputs = graph->inputs();
+  return std::find(inputs.begin(), inputs.end(), value) != inputs.end();
+}
```

Comment:
This might be eligible to be an API in `ir.h`, e.g. `Value::isOwningGraphInput()`

Reply:
we can move this to Value is more use cases comes up :)

Reply:
I think that makes sense, but another thing to consider is that someone might implement basically the same thing and they wouldn't know to talk to us or look in this specific file. So putting it there preemptively ensures new callsites use a common API. Not high-pri though

Reply:
Yeah, that's true. I'm just not sure how useful this function is, if there is one more use case then it justifies us putting this into `ir.h` I guess.